src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -312,7 +312,8 @@
                     "weight": "Weight: The weighting of a test case determines the points a student receives for passing the test case. The number of points a test case awards is proportional to its weight and calculated as a fraction of the total exercise points.",
                     "bonusMultiplier": "Bonus multiplier: This multiplies the points given for passing a test case without affecting the fractions of other test cases. Multipliers greater than 1 can result in student scores greater than 100%.",
                     "bonusPoints": "Bonus points: Additional points added on top of the regular points for passing a test case. Setting bonus points can result in student scores greater than 100%.",
-                    "visibility": "Feedback and points for test cases marked as ‘After Due Date’ only become visible to students after the due date has passed. Tests with visibility ‘Never’ do not influence the student score.",
+                    "visibility": "Feedback and points for test cases marked as ‘{{ afterDueDate }}’ only become visible to students after the due date has passed. Tests with visibility ‘Never’ do not influence the student score.",
+                    "visibilityExam": "Feedback and points for test cases marked as ‘{{ afterReleaseDateOfResults }}’ only become visible to students after the exam results release date has passed. Tests with visibility ‘Never’ do not influence the student score.",
```

Comment:
Why is there a translation for afterReleaseDateOfResults but not for ‘Never’?

Reply:
No particular reason, because I did only change the part directly addressed by this PR for now.

Do you have an opinion on which approach is better, as I also had a discussion with @Strohgelaender  about this https://github.com/ls1intum/Artemis/pull/8451#discussion_r1573838850? Translation (less error prone) vs. hardcoded (less complex)?

Reply:
It is now simplified https://github.com/ls1intum/Artemis/pull/8451/commits/fbe5708e89225f088c1e3e7f2bef0a8cd03cdcdc